examples/active-active/outputs.tf

```diff
@@ -0,0 +1,45 @@
+output "active_active" {
+  value = module.active_active
+
+  description = "The outputs of the active_active module."
+  # This output is marked as sensitive to work around a bug in Terraform 0.14
```

Comment:
Do you know if there is a link to this bug that you can add?

Reply:
Not sure which one - I don't have the link

Reply:
I will do a test without marking output as sensitive and will remove this comment if the bug is fixed

Reply:
Here is the error : 

│ Error: Output refers to sensitive values
│ 
│   on outputs.tf line 30:
│   30: output "tfe_autoscaling_group_name" {
│ 
│ To reduce the risk of accidentally exporting sensitive data that was intended to be only internal, Terraform requires that any root module output containing sensitive data be explicitly marked as sensitive, to confirm your intent.
│ 
│ If you do intend to export this data, annotate the output value as sensitive by adding the following argument:
│     sensitive = true